modules/kubernetes-addons/argocd/argocd-application/helm/templates/application.yaml

```diff
@@ -11,11 +11,16 @@ spec:
     repoURL: {{ .Values.source.repoUrl }}
     targetRevision: {{ .Values.source.targetRevision }}
     path: {{ .Values.source.path }}
-    helm:
-      valueFiles:
-        - 'secrets://values.yaml'
-        - 'secrets://values.ci.yaml'
-      values: {{ toYaml .Values.source.helm.values | toString | indent 6 }}
+    {{ if ((.Values.source).plugin).name -}}
+      plugin:
+        name: {{ .Values.source.plugin.name }}
```

Comment:
I would like to modify implementation, instead of checking for the plugin to be defined you modify for the presence of the **value_files** attr in this configuration
![image](https://github.com/user-attachments/assets/5e4216ca-cde7-431f-809c-ccc632073d15)

and then use if/else

Reply:
so I would say this plugin should be always enabled.

Reply:
sp no need to verify anything